packages/server/src/instances/lifetime/service.ts

```diff
@@ -7,6 +8,8 @@ import { StatusService } from '../../status/service'
 import { InstanceLifetimeDispatcher, InstanceLifetimeDispatches } from './dispatch'
 import { InstanceLifetimeEmitter, InstanceLifetimeEvents, InstanceLifetimeWatcher } from './events'
 
+const MAX_ALLOWED_FAILURES = 5
```

Comment:
This exact value is already defined in the monitoring service. Maybe we should consider exporting it instead of duplicating it? Unless we want to be able to use different values here than the monitoring service?